src/theme.tsx

```diff
@@ -210,38 +210,38 @@ export type EventFormat = 'universal-analytics' | 'google-analytics-4';
 
 // This is the type expected by ThemeContextProvider theme prop.
 // This config is provided by the user of the lib
-export type ThemeConfig = {
+export type ThemeConfig = Readonly<{
     skin: Readonly<Skin>;
     colorScheme?: ColorScheme; // light by default. TODO: Change to auto by default in next major version
-    i18n: {
+    i18n: Readonly<{
         locale: Locale;
         phoneNumberFormattingRegionCode: RegionCode;
-    };
-    platformOverrides?: {
+    }>;
+    platformOverrides?: Readonly<{
         platform?: 'ios' | 'android' | 'desktop';
         insideNovumNativeApp?: boolean;
         userAgent?: string;
-    };
-    texts?: Partial<ThemeTexts>;
-    analytics?: {
+    }>;
+    texts?: Partial<Readonly<ThemeTexts>>;
```

Comment:
Instead of this, type `ThemeTexts` as `Readonly`

Reply:
webapp:
![image](https://user-images.githubusercontent.com/579417/201066396-aff18ab7-34a9-4737-9a95-cabe9ab0b376.png)